examples/js/loaders/sea3d/SEA3DLoader.js

```diff
@@ -2289,115 +1999,129 @@ THREE.SEA3D.prototype.getVertexAnimation = function(sea) {
-	//console.log("SEA3D:", args.object.name + "." + args.object.type);
+	args.file.dispatchEvent( args );
+
 }
 
 THREE.SEA3D.prototype.onError = function( args ) {
+
 	args.file = this.scope; args.type = THREE.SEA3D.Event.ERROR;
-	args.file.dispatchEvent(args);
-	//console.log("SEA3D:", args.message);
+	args.file.dispatchEvent( args );
+
 }
 
 //
 //	Loader
 //
 
-THREE.SEA3D.prototype.load = function( url ) {			
+THREE.SEA3D.prototype.load = function( url ) {
+
 	this.loadBytes();
-	this.file.load(url);		
+	this.file.load( url );
+
+}
+
+THREE.SEA3D.prototype.onHead = function( args ) {
+
+	if ( args.sign != 'TJS' ) {
+
+		throw "Sign '" + args.sign + "' not supported! Use SEA3D Studio to export."
+
+	}
+
 }
 
-THREE.SEA3D.prototype.loadBytes = function( data ) {			
-	this.file = new SEA3D.File( data );
+THREE.SEA3D.EXTENSIONS = [];
+
+THREE.SEA3D.prototype.loadBytes = function( data ) {
+
+	this.file = new SEA3D.File();
 	this.file.scope = this;
 	this.file.onComplete = this.onComplete;
 	this.file.onProgress = this.onLoadProgress;
 	this.file.onCompleteObject = this.onCompleteObject;
 	this.file.onDownloadProgress = this.onDownloadProgress;
 	this.file.onError = this.onError;
-	
-	//	SEA3D
-	
-	switch(this.config.parser)
+	this.file.onHead = this.onHead;
+
+	// SEA3D
+
+	this.file.typeRead[ SEA3D.Geometry.prototype.type ] =
+	this.file.typeRead[ SEA3D.GeometryDelta.prototype.type ] = this.readGeometryBuffer;
+	this.file.typeRead[ SEA3D.Mesh.prototype.type ] = this.readMesh;
+	this.file.typeRead[ SEA3D.Mesh2D.prototype.type ] = this.readMesh2D;
+	this.file.typeRead[ SEA3D.Container3D.prototype.type ] = this.readContainer3D;
+	this.file.typeRead[ SEA3D.Dummy.prototype.type ] = this.readDummy;
+	this.file.typeRead[ SEA3D.Line.prototype.type ] = this.readLine;
+	this.file.typeRead[ SEA3D.Material.prototype.type ] = this.readMaterial;
+	this.file.typeRead[ SEA3D.PointLight.prototype.type ] = this.readPointLight;
+	this.file.typeRead[ SEA3D.DirectionalLight.prototype.type ] = this.readDirectionalLight;
+	this.file.typeRead[ SEA3D.HemisphereLight.prototype.type ] = this.readHemisphereLight;
+	this.file.typeRead[ SEA3D.Camera.prototype.type ] = this.readCamera;
+	this.file.typeRead[ SEA3D.SkeletonLocal.prototype.type ] = this.readSkeletonLocal;
+	this.file.typeRead[ SEA3D.JointObject.prototype.type ] = this.readJointObject;
+	this.file.typeRead[ SEA3D.CubeMap.prototype.type ] = this.readCubeMap;
+	this.file.typeRead[ SEA3D.CubeRender.prototype.type ] = this.readCubeRender;
+	this.file.typeRead[ SEA3D.Animation.prototype.type ] = this.readAnimation;
+	this.file.typeRead[ SEA3D.SoundPoint.prototype.type ] = this.readSoundPoint;
+	this.file.typeRead[ SEA3D.TextureURL.prototype.type ] = this.readTextureURL;
+	this.file.typeRead[ SEA3D.Morph.prototype.type ] = this.readMorpher;
+	this.file.typeRead[ SEA3D.VertexAnimation.prototype.type ] = this.readVertexAnimation;
+
+	// UNIVERSAL
+
+	this.file.typeRead[ SEA3D.JPEG.prototype.type ] =
```

Comment:
I think he refers to the missing rvalue on some of the assignments.

If your goal is to assign the same value to multiple dictionary elements by chaining, eg `a = b = c = 3`, I am not sure its safe to do so with linebreaks in JavaScript since a `;` could be inserted eol if im not misstanken.